test/index.test.js

```diff
@@ -729,4 +729,20 @@ describe('test fetch retry', () => {
         const response = await fetch(`${FAKE_BASE_URL}${FAKE_PATH}`);
         assert.strictEqual(response.ok, true);
     });
+    
+    it('doesn\'t modify the passed in options', async () => {
```

Comment:
Switch to double-quotes, so you don't need the escaping:
```suggestion
    it("doesn't modify the passed in options", async () => {
```